commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/MathUtilities.scala

```diff
@@ -99,19 +99,22 @@ object MathUtilities {
     }
   }
 
-  def limitCurrentOutput(input: Dimensionless, normalizedV: Dimensionless,
-                         forwardCurrentLimit: Dimensionless,
-                         backwardsCurrentLimit: Dimensionless): Dimensionless = {
-    if(normalizedV < Each(0)) {
-      -limitCurrentOutput(-input, -normalizedV, forwardCurrentLimit, backwardsCurrentLimit)
-    }
-    if(input > normalizedV){
-      input.min(normalizedV + forwardCurrentLimit)
-    } else if(input < Each(0)){
-      val limitedInput = Each(-backwardsCurrentLimit / (Each(1) + normalizedV))
-      limitedInput.max(input)
+  /**
+    * swapps a trigometric angle, as used in the mathematics
+    * (right is 0 degrees, increasing counter clockwise) to
+    * compass angle, forward is 0, increases clockwise, or vice versa
+    * @param trigonemtricAngle
+    * @return
+    */
+  def swapTrigonemtricAndCompass(trigonemtricAngle: Angle): Angle = {
+    Degrees(90) - trigonemtricAngle
+  }
+
+  def clamp[T](toClamp: Dimensionless, max: Dimensionless): Dimensionless = {
```

Comment:
At some point we could probably add this as a feature to Squants! `quantity.clamp(range)` or something like that for more generic support of clamping.